report.tex

```diff
@@ -15,6 +24,21 @@
 \bibliographystyle{plain}	% Literaturverzeichnisformat = standard
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+%OTHER OPTIONS
+\setlength{\parindent}{0px} %dieser Befehl verhindert das Einrücken eines Absatzes bei einer leeren Codezeile. 
+
+%SHORTCUTS
+\newcommand{\R}{\mathbb{R}} %reelle Zahlen
+\newcommand{\N}{\mathbb{N}} %natürliche Zahlen
+\newcommand{\Z}{\mathbb{Z}} %ganze Zahlen
+\newcommand{\C}{\mathbb{C}} %komplexe Zahlen
+
+\newcommand{\stackeq}[1]{\mathrel{\stackrel{\makebox[0pt]{\mbox{\normalfont\tiny #1}}}{=}}} %das ist ein Gleichheitszeichen mit Text darüber, Beispiel: $a\stackeq{Def} b$
```

Comment:
As the newcommand below defines a more generic approach, this can be omitted.

Reply:
Or at least use the command below to define this one. Code duplication!

Reply:
Right! I'll fix this.